hooks/pre-exit

```diff
@@ -4,16 +4,16 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 # shellcheck disable=SC1091
 . "${DIR}/../lib/setup.bash"
 
-# shellcheck disable=SC2086
-REGISTRY_HOSTNAME=$(vault $VAULT_METHOD -field=hostname "$BUILDKITE_PLUGIN_VAULT_DOCKER_LOGIN_SECRET_PATH")
-
-echo "~~~ Log out to $REGISTRY_HOSTNAME container registry"
-
 if [[ "${BUILDKITE_PLUGIN_VAULT_DOCKER_LOGIN_DISABLE_LOGOUT:-false}" = "true" ]]; then
   echo 'WARNING: not logging out'
```

Comment:
I think ideally we'd want this line to be below the "~~~" line that marks this "section" in the Buildkite output. That's harder though because you'd have to duplicate the `if [[ "${BUILDKITE_PLUGIN_VAULT_DOCKER_LOGIN_DISABLE_LOGOUT:-false}" = "true" ]]` check. What do you think makes the most sense to fix this?